practice-app/server/src/services/db.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require("mongoose");
```

Comment:
I think we do not need to export db object to other files. All the communication between backend and db should be via mongoose.

Reply:
removed file.